config/application.rb

```diff
@@ -15,5 +15,12 @@ class Application < Rails::Application
     # Application configuration can go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded after loading
     # the framework and any gems in your application.
+
+    config.after_initialize do
+      # Set QuickSearch configs to values in our Settings files
+      # (so we can handle them on an env by env basis in our standard way)
+      QuickSearch::Engine::APP_CONFIG['searchers'] = Settings.ENABLED_SEARCHERS
+      QuickSearch::Engine::APP_CONFIG['found_types'] = Settings.ENABLED_SEARCHERS
```

Comment:
I'm pretty `meh` on this, but otherwise we don't really have a good way of setting these things in a way that we're used to (e.g. in Settings that can be enabled in tests and on non-prod servers).

If this seems legit, we can always split these into two configs if they differ (but I'm not sure I see a reason why they would).